lib/internal/Magento/Framework/View/Helper/SecureHtmlRenderer.php

```diff
@@ -111,16 +111,21 @@ public function renderEventListenerAsTag(
             function {$listenerFunction} () {
                 {$attributeJavascript};
             }
-            var {$elementName} = document.querySelector("{$elementSelector}");
-            if ({$elementName}) {
-                {$elementName}.{$eventName} = function (event) {
-                    var targetElement = {$elementName};
-                    if (event && event.target) {
-                        targetElement = event.target;
+            var {$elementName}Array = document.querySelectorAll("{$elementSelector}");
+            if({$elementName}Array.lenght !== 'undefined'){
+                {$elementName}Array.forEach(function(element){
```

Comment:
Should change Array.lenght  to Array.length in line 115
@rogyar @saphaljha

Reply:
Hello @mrtuvn ,
Thank you, I have updated it.